Mathlib/Algebra/Module/Submodule/Lattice.lean

```diff
@@ -348,6 +350,35 @@ theorem mem_sSup_of_mem {S : Set (Submodule R M)} {s : Submodule R M} (hs : s 
   exact this
 #align submodule.mem_Sup_of_mem Submodule.mem_sSup_of_mem
 
+-- TODO Change `Submodule.completeLattice` so that this is true by definition.
```

Comment:
I have actually made this lemma true by definition in #9695 but it's not yet clear that we can accept the performance consequences there.